data/release/wmts.xml

```diff
@@ -0,0 +1,22 @@
+<wmts>
+  <id>WMTSInfoImpl--3ef31051:182c1e8b194:-8000</id>
+  <enabled>true</enabled>
+  <name>WMTS</name>
+  <title>GeoServer Web Map Tile Service</title>
+  <maintainer>http://geoserver.org/com</maintainer>
+  <abstrct>A compliant implementation of WMTS service.</abstrct>
```

Comment:
Possible typo? Or do we really do this?

```suggestion
  <abstract>A compliant implementation of WMTS service.</abstract>
```

Reply:
We really do it :rofl: ... this data structure is a dump of a Java class properties, and "abstract" is a reserved keyword...